Installing Ember fails, specified version can't be found

Quick start ( https://guides.emberjs.com/v2.6.0/getting-started/quick-start/ ) currently recommends this:

```
npm install -g ember-cli@2.6
```

That immediately failed with this error:
```
$ npm install -g ember-cli@2.6
npm ERR! Darwin 15.4.0
npm ERR! argv "/Users/jzaefferer/.nvm/versions/node/v5.3.0/bin/node" "/Users/jzaefferer/.nvm/versions/node/v5.3.0/bin/npm" "install" "-g" "ember-cli@2.6"
npm ERR! node v5.3.0
npm ERR! npm  v3.9.3

npm ERR! No compatible version found: ember-cli@2.6
npm ERR! Valid install targets:
npm ERR! 2.6.0-beta.3, 2.6.0-beta.2, 2.6.0-beta.1, 2.5.1, 2.5.0, 2.4.3, 2.4.2, 2.4.1, 2.4.0, 2.3.0, 2.3.0-beta.2, 2.3.0-beta.1, 2.2.0-beta.6, 2.2.0-beta.5, 2.2.0-beta.4, 2.2.0-beta.3, 2.2.0-beta.2, 2.2.0-beta.1, 1.13.15, 1.13.14, 1.13.13, 1.13.12, 1.13.11, 1.13.10, 1.13.9, 1.13.8, 1.13.7, 1.13.6, 1.13.5, 1.13.1, 1.13.0, 0.2.7, 0.2.6, 0.2.5, 0.2.4, 0.2.3, 0.2.2, 0.2.1, 0.2.0, 0.2.0-beta.1, 0.1.15, 0.1.14, 0.1.13, 0.1.12, 0.1.11, 0.1.10, 0.1.9, 0.1.8, 0.1.7, 0.1.6, 0.1.5, 0.1.4, 0.1.3, 0.1.2, 0.1.1, 0.1.0, 0.0.47, 0.0.46, 0.0.45, 0.0.44, 0.0.43, 0.0.42, 0.0.41, 0.0.40, 0.0.39, 0.0.38, 0.0.37, 0.0.36, 0.0.35, 0.0.34, 0.0.33, 0.0.32, 0.0.29, 0.0.28, 0.0.27, 0.0.25, 0.0.24, 0.0.23, 0.0.22, 0.0.21, 0.0.20, 0.0.19, 0.0.18, 0.0.16, 0.0.15, 0.0.14, 0.0.13, 0.0.12, 0.0.11, 0.0.10, 0.0.9, 0.0.8, 0.0.7, 0.0.6, 0.0.5, 0.0.4, 0.0.3, 0.0.2, 0.0.1, 0.0.0
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jzaefferer/dev/npm-debug.log
```
Apparently the guide recommends installing a beta version, which doesn't work like that. 

I went with `$ npm install -g ember-cli@latest` instead. Not sure yet what that installs (still working), I guess it'll be 2.5.1. (yep, it is)

--- a/source/javascripts/app/version-select.js.erb
+++ b/source/javascripts/app/version-select.js.erb
@@ -14,8 +14,8 @@ window.GUIDE_VERSIONS.onReady(function (versions) {
   });
 
   var npmInstallVersion = $('pre:contains("install -g ember-cli")');
-
-  if (currentRevision.slice(0,1) !== "C") {
+  //only append the version if there's one specified and not one already in the content
+  if (currentRevision.slice(0,1) !== "C" && npmInstallVersion.text().indexOf("@") === -1) {
     var installVersion = versions.current.slice(1).split(".").slice(0,2).join(".");
 
     npmInstallVersion.text("npm install -g ember-cli@" + installVersion);

--- a/source/localizable/getting-started/index.md
+++ b/source/localizable/getting-started/index.md
@@ -55,7 +55,7 @@ need for a browser to be open. Consult the [PhantomJS download instructions](htt
 Install Ember using npm:
 
 ```bash
-npm install -g ember-cli
+npm install -g ember-cli@2.5.1
 ```
 
 To verify that your installation was successful, run:

--- a/source/localizable/getting-started/quick-start.md
+++ b/source/localizable/getting-started/quick-start.md
@@ -15,7 +15,7 @@ You can install Ember with a single command using npm, the Node.js package
 manager. Type this into your terminal:
 
 ```sh
-npm install -g ember-cli
+npm install -g ember-cli@2.5.1
 ```
 
 Don't have npm? [Learn how to install Node.js and npm here][npm].

